ui/typings/JQuery.ts

```diff
@@ -0,0 +1,68 @@
+interface JQuery {
+    pulsate(action: string | PulsateOptions): void;
+
+    datetimepicker(options?: DateTimePickerOptions): JQuery;
+
+    multiSelect(option: string | MultiSelectOptions, options?: MultiSelectOptions | any): void;
+}
+
+interface DateTimePickerOptions {
+    format?: string;
+    timepicker?: boolean;
+}
+
+interface JQueryStatic {
+    confirm(options?: ConfirmOptions): void;
+
+    alert(options?: AlertOptions | string | Alert): void;
+}
+
+interface MultiSelectOptions {
+    value?: string;
+    symbol?: string;
+    selectableHeader?: string;
+    selectionHeader?: string;
+    afterSelect?: (values: any) => void;
+    afterDeselect?: (values: any) => void;
+    afterInit?: (container: any) => void;
+}
+
+
+interface AlertOptions {
+    content: () => void;
+}
+
+interface Alert {
+    buttons?: { [name: string]: { action: () => void } };
+
+    setType(type: string): void;
+
+    setTitle(title: string): void;
+
+    setContentAppend(content: string): void;
+}
+
+interface PulsateOptions {
+    color?: string;
+    reach?: number;                              // how far the pulse goes in px
```

Comment:
nitpick: Can you make the comments all start at the same line and be aligned to each other?